tests/lie_tests/functional/test_so3.py

```diff
@@ -7,7 +7,7 @@
 import pytest
 import torch
 
-from tests.decorators import run_if_labs
+from tests.theseus_tests.decorators import run_if_labs
```

Comment:
Do we still need `run_if_labs` if `torchlie` has been moved out of labs?

Reply:
We don't need it, but I had missed this at first. I removed them in later PR, [here](https://github.com/facebookresearch/theseus/pull/527/commits).